dataland-frontend/tests/component/components/forms/parts/fields/CurrencyDataPointFormField.cy.ts

```diff
@@ -7,10 +7,7 @@ describe('Component test for CurrencyDataPointFormField', () => {
    * Ensures that the data quality field is inactive and that the field is not shown as required with an asterix
    */
   function checkDataQualityFieldNotActive(): void {
-    cy.get('div[data-test="dataQuality"] div[name="quality"]')
-      .should('have.class', 'p-disabled')
-      .find('span')
-      .should('have.text', 'Data quality');
+    cy.get('div[data-test="dataQuality"] div[name="quality"]').find('span').should('have.text', 'Data quality');
```

Comment:
Jetzt tut diese Funktion nicht mehr das, was sie im Namen behauptet zu tun.

Vorher hat sie getestet, ob das Dropdown für die data-quality ausgegraut ist (`p-disabled`).
Das war vorher der Fall, wenn man noch keinen `value` für den data-point eingetragen hatte.

Jetzt ist das Dropdown durch deine changes in `ExtendedDataPointFormField` immer verfügbar (und nie ausgegraut).
Daher macht die Funktion eigentlich keinen Sinn mehr bzw. sie testet bloß die Existenz des Dropdowns für die quality.

Reply:
Docstring stimmt jetzt nicht mehr und der Test-Code ist teilweise sinnlos, nachdem das Ausgrauen des Data-Quality-Felds nicht mehr an den value gekoppelt ist.

Ich habe das angepasst. Das ist der commit:

https://github.com/d-fine/Dataland/commit/7f9074f23abc1b9bfbcfff52fecae166a462f860


Wenn das für dich ok ist, schließe ich die conversation.